vector/src/main/res/values/strings.xml

```diff
@@ -2245,6 +2249,7 @@
     <string name="push_gateway_item_device_name">session_name:</string>
     <string name="push_gateway_item_url">Url:</string>
     <string name="push_gateway_item_format">Format:</string>
+    <string name="push_gateway_item_remove">Remove</string>
```

Comment:
Maybe use the existing string R.string.remove (which should be renamed in the future with `action_remove`, for name coherency with other common actions), to avoid too many string duplication.

Reply:
43eff8b85bbfa4c20f21204c9d3d05444b3b1770